pallets/operation/src/lib.rs

```diff
@@ -397,6 +403,72 @@ pub mod pallet {
             Self::deposit_event(Event::<T>::NpowMint(target, dpr));
             Ok(().into())
         }
+
+        #[pallet::weight(10_000 + T::DbWeight::get().reads_writes(1,1))]
+        pub fn set_fund_pool_address(
+            origin: OriginFor<T>,
+            funder: T::AccountId,
+        ) -> DispatchResultWithPostInfo {
+            let who = ensure_signed(origin)?;
+            ensure!(
+                T::UserPrivilegeInterface::has_privilege(&who, Privilege::BridgeAdmin),
+                Error::<T>::UnauthorizedAccounts
+            );
+            BridgeFundAddreess::<T>::put(funder);
+            Ok(().into())
+        }
+
+        #[pallet::weight(T::OPWeightInfo::bridge_deeper_to_other())]
+        pub fn bridge_deeper_to_other(
```

Comment:
What's the purpose of this function, if want to transfer money to `H160` address, you can try convert H160 to AccountId

Reply:
H160  is used only for display in event

Reply:
Still not understand why use don't use `pallet-evm`'s `into_account_id` to transfer directly.

Reply:
H160 address is not our frontier evm address, it is the address of bsc .(Binance)
this funciton is used for bsc bridge, so we can't  use into_account_id  to transfer